### PR TITLE
UPDATE #212-2: Adjust search button option

### LINKED ISSUE #212: Discussion: Rephrase the Disable Search Button option
This is the current text for the search button option in the customizer

----

Settings for the site header.

- [ ] Disable Search Button

_Check to disable the search button in the header._

----

I think it would make more sense for this option to be reversed;
for the search to be enabled by default, with the box checked,
and disabled when unchecked. 

I don't think the word "Button" is needed.

The text could be rephrased to something like:

----

Settings for the site header.

- [x] Enable search

_Uncheck to disable the search in the header._

----

Currently, the Disable Search Button option is in the site header panel.
**-I keep clicking this panel expecting to add a header image.**
And this is is despite knowing very well that the header image panel has a different name :)

### PR DESCRIPTION
**Pull request by nielslange**
Fixes #212

### CODE REVIEW

File: inc/template-tags.php [hunk 1]
```diff
@@ -457,9 +457,9 @@ function twentytwenty_body_classes( $classes ) {
 			$classes[] = 'has-full-width-content';
 		}
 
-		// Check for disabled search.
-		if ( get_theme_mod( 'twentytwenty_disable_header_search', false ) ) {
-			$classes[] = 'disable-search-modal';
+		// Check for enabled search.
+		if ( true === get_theme_mod( 'twentytwenty_enable_header_search' ) ) {
+			$classes[] = 'enable-search-modal';
```
Comment (joyously) [inline inc/template-tags.php]: Does this class affect any CSS?

Comment (nielslange) [reply inc/template-tags.php]: @ joyously No, currently there is no CSS class `.disable-search-modal`. I believe @andersnoren added that body class to enable users to add conditional CSS, when needed, later on.

Comment (joyously) [reply inc/template-tags.php]: OK, well, I just searched the files for 'search-modal' and the JS uses `search-modal`, which the CSS uses also. The header and the modal-search file both have the `search-modal` class, but also have data attributes 
`data-toggle-target=".search-modal"`
`data-toggle-body-class="showing-search-modal"`
`data-set-focus=".search-modal .search-field"`

Seems like the body class should match. (and 'showing' makes more sense than 'enable')

File: header.php [hunk 2]
```diff
@@ -115,10 +115,10 @@
 
 						<?php
 
-						// Check whether the header search is deactivated in the customizer.
-						$disable_header_search = get_theme_mod( 'twentytwenty_disable_header_search', false );
+						// Check whether the header search is activated in the customizer.
+						$enable_header_search = get_theme_mod( 'twentytwenty_enable_header_search', false );
```
Comment (andersnoren) [inline header.php]: This should be updated to have `true` as default value, to match the `add_control()`.

**Comment by carolinan** [comment]
Sorry about this mess I made :) . -Please also hide the mobile search form, starting on line 40 in header.php.

File: header.php (hunk 2 above)
Comment (nielslange) [reply header.php]: Nice catch, @andersnoren! I've fixed this issue in https://github.com/WordPress/twentytwenty/pull/264

**Comment by nielslange** [comment]
> Please also hide the mobile search form toggle, starting on line 40 in header.php.

@carolinan With https://github.com/WordPress/twentytwenty/pull/264 the visibility of the mobile search icon can be toggled. Feel free to merge this PR again, if you believe it's in a good shape now.

**Pull request merged**